internal/router/router.go

```diff
@@ -170,13 +131,6 @@ func (r *Router) get(uri string, f http.HandlerFunc, mid ...mux.MiddlewareFunc)
 	sub.Use(mid...)
 }
 
-// post creates a subroute on the specified URI that only accepts POST. You can provide specific middlewares.
```

Comment:
nre, having the Validator no longer accept POST requests feels, to me, more conceptually inline with the protocol.